### PR TITLE
Milestone: Created modules

### PR DESCRIPTION
**Pull request by Roseokpe**
In this project, I restructured my Awesome books app code into modules. The goal is to make it more organized by using modules. And also practice the ES6 syntax that I have learnt.

### CODE REVIEW

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Roseokpe,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a Pull Request that has no linters set up.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Roseokpe** [comment]
@TedLivist I hope you're doing great. My project passed all the linters error, cause I checked and fixed them before pushing it.

Linters check are set on my project. Kindly confirm and review my project. Thank you

File: style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Lato&family=Poppins&display=swap');
```
Comment (vechicin) [inline style.css]: Please change this file name to `index.css`.

Screen Shot 2022-07-04 at 9 27 09

File: script.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import submitForm from './modules/submit-form.js';
```
Comment (vechicin) [inline script.js]: Please change this file name to `index.js`.

Screen Shot 2022-07-04 at 9 27 40

File: modules/delete-books.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import getLocalStorageData from './load-store.js';
+
+export default function (key, awesomeBook) {
```
Comment (vechicin) [inline modules/delete-books.js]: Please use arrow functions.

Screen Shot 2022-07-04 at 9 29 59

File: modules/load-store.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import displayBooks from './display-books.js';
+
+export default function (awesomeBook) {
```
Comment (vechicin) [inline modules/load-store.js]: Please use arrow functions.

Screen Shot 2022-07-04 at 9 29 59

File: modules/submit-form.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import getLocalStorageData from './load-store.js';
+
+export default function (title, author, bookList, awesomeBook) {
```
Comment (vechicin) [inline modules/submit-form.js]: Please use arrow functions.

Screen Shot 2022-07-04 at 9 29 59

File: .github/workflow/linters.yml [hunk 6]
```diff
@@ -0,0 +1,70 @@
+name: Linters
```
Comment (vechicin) [inline .github/workflow/linters.yml]: Please change the folder name to `workflows`, that's the reason why GitHub Actions is not picking up on your linters checks.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Roseokpe ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️

- [ ] Please install `stylelint`

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Roseokpe) [reply style.css]: @vechicin please why do you want me to change it?

**Review by Lordkaito — APPROVED** [review]
Hi @Roseokpe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**